spec/latest/core/core.jsonld

```diff
@@ -180,19 +180,19 @@
       "vs:term_status": "testing"
     },
     {
-      "@id": "hydra:readonly",
+      "@id": "hydra:readable",
       "@type": "rdf:Property",
       "label": "ready-only",
-      "comment": "True if the property is read-only, false otherwise.",
+      "comment": "True if the client can read the property, false otherwise.",

```

Comment:
Same as in index.html - but I read this afterwards :-)

"True if *a* client can *retrieve/change* the property's value, false otherwise".